recipe/build.sh

```diff
@@ -1,12 +1,164 @@
 #!/bin/bash
 
-mkdir -p $PREFIX/bin
-mkdir -p $PREFIX/include
-mkdir -p $PREFIX/lib
-mkdir -p $PREFIX/share/man/man1/
+# First build patched qt4 static lib
+cd wkhtmltopdf/qt
 
+# We're linking openssl statically with the -openssl-linked argument
+# to qt's configure. So we need libopenssl-static. We also need the
+# headers from openssl. But if we add those packages to requirements
+# in meta.yml as normal, there will be conflicts and we can't do the
+# following hack to remove the .so files to force it to be linked
+# statically.
+conda create -y --prefix "${SRC_DIR}/openssl_hack" -c conda-forge  \
+      --no-deps --yes --copy --prefix "${SRC_DIR}/openssl_hack"  \
+      libopenssl-static=1.1.* openssl=1.1.*
+export OPENSSL_LIBS="-L${SRC_DIR}/openssl_hack/lib -lssl -lcrypto"
+# Remove the .so files so the linker will actually do static linking.
+rm ${SRC_DIR}/openssl_hack/lib/libcrypto.so*
+rm ${SRC_DIR}/openssl_hack/lib/libssl.so*
+#rm -rf ${PREFIX}/include/openssl
 
-mv bin/* $PREFIX/bin/
-mv include/wkhtmltox/ $PREFIX/include/
-mv lib/* $PREFIX/lib/
-mv share/man/man1/* $PREFIX/share/man/man1/
+# Remainder of this setup before ./configure is cribbed from the qt4 branch of qt-feedstock.
+compiler_mkspec=mkspecs/common/g++-base.conf
+flag_mkspec=mkspecs/linux-g++/qmake.conf
+
+# The Anaconda gcc7 compiler flags specify -std=c++17 by default, which
+# activates features that break compilation. Begone!
+CXXFLAGS=$(echo $CXXFLAGS | sed -E 's@\-std=[^ ]+@@')
+# Deviation from qt-feedstock: wkhtmltopdf uses some gnu extensions.
+export CXXFLAGS="$CXXFLAGS -std=gnu++11"
+
+# This warning causes a huge amount of spew in the build logs.
+if [ "$cxx_compiler" = gxx ] ; then
+CXXFLAGS="$CXXFLAGS -Wno-expansion-to-defined"
```

Comment:
Is there supposed to be an export here?

Reply:
Ah you might be right - this was copied from qt-feedstock. Actually maybe the `export`s aren't required at all - CXXFLAGS is `unset` after being used to modify `$flag_mkspec`?